src/main/scala/se/lu/nateko/cp/data/formats/TimeSeriesParsing.scala

```diff
@@ -1,5 +1,16 @@
 package se.lu.nateko.cp.data.formats
 
+trait TextFormatParser[A <: StandardParsingAcculumator] {
+	def parseLine(acc: A, line: String): A
```

Comment:
(Optional, experimental) Consider changing the design slightly by removing the type parameter from `TextFormatParser` and making it type member `type A <: StandardParsingAcculumator` instead. `TextFormatParser` can then also provide `def seed: A`, which kind of makes sense, and also reduces the number of abstract members in `StandardCsvStreams` by 2.